src/time_gf.jl

```diff
@@ -87,6 +87,11 @@ IndexStyle(::Type{<:TimeGF}) = IndexLinear()
 size(gf::TimeGF) = size(gf.data)
 Base.@propagate_inbounds getindex(gf::TimeGF, i::Int) = gf.data[i]
 Base.@propagate_inbounds setindex!(gf::TimeGF, v, i::Int) = gf.data[i] = v
+
+Base.@propagate_inbounds Base.getindex(gf::TimeGF, i::Int, j::Int) = view(gf.data[:, :, i, j])
```

Comment:
should be `view(gf.data, :, :, i, j)`

Reply:
Thank you for the pointer! Fixed, below :)